lib/html/template/rose-bush/index.html

```diff
@@ -12,6 +12,9 @@
 <body>
 <div class="container-fluid">
 <div class="hero-unit">
+{% if logo -%}
+<img {{logo}} />
```

Comment:
I did consider an alternate approach, which will break the attributes into key-value pairs, but it would require a bit more logic. The implemented approach requires the least logic, but is just as effective.